### ISSUE TITLE
[import/named]: export * as namespace 

### ISSUE THREAD

**Issue by yoyo837** [import-js/eslint-plugin-import#1998]
models:
```js
export * as ExtfieldModel from './extfield';
export * as Extfield2Model from './extfield2';
```

usage:
```js
import { ExtfieldModel, Extfield2Model } from './models';
```

It report:
`ExtfieldModel not found in './models' eslint(import/named)`

**Comment by brunoti** [comment]
Same problem here

**Comment by ljharb** [comment]
Released in v2.24.1

**Comment by yoyo837** [comment]
Thanks for your hard work.
